src/views/StoryTemplate.js

```diff
@@ -139,43 +48,42 @@ const QuestionActions = styled.div`
   margin-bottom: 40px;
 `;
 
-const StoryTemplate = () => (
-  <Fragment>
-    <Helmet>
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
-      <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
-    </Helmet>
+class StoryTemplate extends Component {
```

Comment:
Is there a specific reason why you converted this into a stateful component?

Reply:
This was to be able to change the state when an input is given. I can change it back if this is not correct.